src/main/java/at/feldim2425/moreoverlays/lightoverlay/LightOverlayRendererZharkCustomSpawner.java

```diff
@@ -0,0 +1,29 @@
+package at.feldim2425.moreoverlays.lightoverlay;
+
+import java.util.List;
+
+import drzhark.customspawner.CustomSpawner;
+import drzhark.customspawner.environment.EnvironmentSettings;
+import drzhark.customspawner.utils.CMSUtils;
```

Comment:
If you import classes from this mod is there some repo where gradle can download it for the build?